packages/salesforcedx-vscode-core/src/commands/forceSourceDeploySourcePath.ts

```diff
@@ -4,7 +4,7 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import { ContinueResponse } from '@salesforce/salesforcedx-utils-vscode';
+import { ContinueResponse, workspaceUtils } from '@salesforce/salesforcedx-utils-vscode';
```

Comment:
The additional import of `workspaceUtils` is unused.  Also, is there a reason this file is not renamed to `projectDeployStartSourcePath.ts`?

Reply:
good catch for the workspaceUtils! This file is used for deploy commands, so this is not in the scope of this PR